rusk/tests/services/prover_service.rs

```diff
@@ -84,31 +83,76 @@ impl Store for TestStore {
 }
 
 #[derive(Debug)]
-struct TestNodeClient {
-    client: Mutex<ProverClient<Channel>>,
+struct TestWalletProverClient {
+    _client: Mutex<ProverClient<Channel>>,
+}
+
+impl TestWalletProverClient {
+    fn new(client: ProverClient<Channel>) -> Self {
+        Self {
+            _client: Mutex::new(client),
+        }
+    }
+}
 
+impl WalletProverClient for TestWalletProverClient {
+    type Error = ();
+
+    /// Requests that a node prove the given transaction and later propagates it
+    fn compute_proof_and_propagate(
+        &self,
+        _utx: &UnprovenTransaction,
+    ) -> Result<(), Self::Error> {
+        unimplemented!();
```

Comment:
Maybe track with an issue?

Reply:
This would be solved by @herr-seppia PR, so it's already tracked.